config/routes.rb

```diff
@@ -58,6 +58,8 @@
     PROJECT_CONSTRAINT = /[^\/]+/
     VERSION_CONSTRAINT = /[\w\.\-]+/
 
+    get '/priority/:platform/:name/:priority', to: 'project_update_priority#update', constraints: { :platform => PLATFORM_CONSTRAINT, :name => PROJECT_CONSTRAINT }
```

Comment:
Should this be a `post` or `put` instead of `get`?

Reply:
Also I think we should consider a batch input for this instead of calling these one by one.